README.md

```diff
@@ -65,6 +65,15 @@ Only `linux-amd64-libcxx-libinstana_sensor.so` is required.
 
 ## Release History
 
+### 1.11.0 (2025-01-21)
+
+  * Add support for envoy-proxy 1.32.2 1.32.3 and 1.33.0
+  * Removed the agent header check for the `Server` header in the discovery response.
```

Comment:
Maybe let's use the same style as the docs team uses in https://www.ibm.com/docs/en/instana-observability/current?topic=tracer-cpp-sensor-nginx-envoy
```suggestion
  * The discovery response is not checked for the agent header `Server` anymore.
```